recipes/openai_ner.py

```diff
@@ -256,7 +248,8 @@ def _get_ner_response(self, prompts: List[str]) -> List[str]:
         responses = r.json()
         return [responses["choices"][i]["text"] for i in range(len(prompts))]
 
-    def _parse_response(self, text: str) -> List[Tuple[str, List[str]]]:
+    @classmethod
+    def parse_response(cls, text: str) -> List[Tuple[str, List[str]]]:
```

Comment:
I made this a classmethod so I could use it as a helper without instantiating the class.

I didn't want to just make it a loose function, because the response parsing will depend on the template people use, so it's reasonable to expect people to want to subclass for this.